spec/spec_helper.rb

```diff
@@ -7,6 +7,7 @@
 SPEC_ROOT           = Pathname.new File.expand_path File.dirname __FILE__
 AR_LOG              = SPEC_ROOT.join('debug.log').to_s
 ENCRYPTION_PASSWORD = "supermadsecretstring"
+FALLBACK_ENCRYPTION_PASSWORD = "fallbacksecretstring"
```

Comment:
can you align the assignment?
the above ones are aligned

Reply:
Fixed